### PR TITLE
Fix test run chrome addon intermittent failure

### PR DESCRIPTION
**Pull request by rpl**
This PR reduces the number of intermittent failures (and help with #374) by tweaking the test case *"jpm run addons - chrome"*:
- retrying panel.show until it succeed.

### CODE REVIEW

File: test/addon_runners/chrome/main.js [hunk 1]
```diff
@@ -84,7 +90,13 @@ exports.testChromeInPanel = function(assert, done) {
     });
     panel.port.emit('echo');
   });
-  panel.show();
+
+  // NOTE: hack needed to prevents random test timeout (probably due to the
+  // setTimeout used in the 'panel.show' method:
+  // https://github.com/mozilla/addon-sdk/blob/a19bd8c/lib/sdk/panel/utils.js#L78)
+  intervalId = setInterval( _ => {
```
Comment (kumar303) [inline test/addon_runners/chrome/main.js]: looks like a good enough approach! :) My only worry is that if `show()` is never called then this interval would live on even if the test times out. Or will it die along with the timeout? If there's no way to address that then no worries, but here are some ideas:

* is there a `afterEach()` or `tearDown()` hook that can be used here to clear the interval?
* should the interval give up after several tries?

**Comment by kumar303** [comment]
My comments are more food-for-thought and could be addressed in follow-up commits. This looks like a decent approach to addressing the timeouts so I'll go ahead and merge it as is. Thanks!

File: test/addon_runners/chrome/main.js (hunk 1 above)
Comment (rpl) [inline test/addon_runners/chrome/main.js]: @kumar303 this test case actually runs inside the Firefox instance launched by the jpm's mocha test case and it is not reused for running the other test addons in the jpm test suite.

In any case it is absolutely reasonable to cleanup the setInterval after the test timeouts.

I'm going to add it in a follow-ups.

**Comment by rpl** [comment]
@kumar303 I agree absolutely, there are  a couple of more details that I would like to look into to completely solve the issue.

And in the mean time we will be able to see if the Travis builds are more stable on building the next pull requests

**Comment by kumar303** [comment]
@rpl I've been watching the builds and they do seem more stable :) I have seen at least one other timeout, which in my haste I did not make note of. It was not in the test you addressed here though IIRC. It went away after a retry.

**Pull request merged**